source/math/FloatMatrix.ooc

```diff
@@ -51,7 +51,8 @@ FloatMatrix : cover {
 	// <param name="x">Column number of a matrix.</param>
 	// <param name="y">Row number of a matrix.</param>
 	// <returns></returns>
-	get: func@ (x, y: Int) -> Float { this elements[x + this width * y] }
+	get: func@ (x, y: Int) -> Float { this elements[x + this width * y] } //TODO Deprecated, remove when no longer used
+	operator [] (x, y: Int) -> Float { this elements[x + dimensions width * y] }
```

Comment:
Why not use `this width`, as above, instead of `dimensions width` (which, by the way, should be preceded by `this `)?

Actually, there's no point in implementing the same function twice. Make the operators call the `get` and `set` functions or vice versa, and when we've updated all callers to use the operators, remove the `get` and `set` functions entirely.